packages/@orchidui-vue/src/Elements/AdditionalContent/OcAdditionalContent.vue

```diff
@@ -53,16 +53,16 @@ const copyLink = async () => {
       :title="additionalTitle"
       class="border rounded md:py-4 py-3 md:px-5 px-4 border-oc-gray-200 bg-oc-gray-50 flex flex-col gap-y-2"
     >
-      <template #title="{ title }">
+      <template #title>
         <div class="flex items-center gap-x-3">
-          <span class="font-medium">{{ title }}</span>
+          <span class="font-medium">{{ additionalTitle }}</span>
           <Chip :variant="chipVariant" :label="chipLabel" />
         </div>
       </template>
-      <template #description="{ description }">
+      <template #description>
```

Comment:
Same here for `description` prop in `OcTitle`

Reply:
Same here with the description slot:
`<slot name="description" :description="description">`

Reply:
ocTitle component has been updated and doesn't support scoped slots, so I removed the slot props.

Reply:
Sorry my bad. Was looking at an outdated version of OcTitle